source/stylesheets/refills/_icon-bullet-points.scss

```diff
@@ -66,10 +41,22 @@
   }
 
   h2 {
-    border-bottom: 1px solid transparentize($base-font-color, 0.8);
+    border-bottom: 1px solid transparentize($base-font-color-on-background, 0.8);
     display: inline-block;
     font-size: $icon-bullet-size / 2.5;
-    margin-bottom: $icon-bullet-size / 6;
+    margin: 0 0 ($icon-bullet-size / 6) 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 $icon-bullet-size / 6` instead of `0 0 $icon-bullet-size / 6 0`